### PR TITLE
[Doc]add st engine to start v2

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+```shell
+export version="2.3.0-beta"
+wget "https://archive.apache.org/dist/incubator/seatunnel/${version}/apache-seatunnel-incubating-${version}-bin.tar.gz"
+tar -xzvf "apache-seatunnel-incubating-${version}-bin.tar.gz"
+```
+<!-- TODO: We should add example module as quick start which is no need for install Spark or Flink -->
+
+## Step 3: Install connectors plugin
+Since 2.3.0-beta, the binary package does not provide connector dependencies by default, so when using it for the first time, we need to execute the following command to install the connector: (Of course, you can also manually download the connector from [Apache Maven Repository](https://repo. maven.apache.org/maven2/org/apache/seatunnel/ to download, then manually move to the seatunnel subdirectory under the connectors directory).
```
Comment (TaoZex) [inline versioned_docs/version-2.3.0-beta/start-v2/local.mdx:37]: Remove the space in the link

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+```plugin_config
+--seatunnel-connectors--
+connector-fake
+connector-console
+--end--
+```
+
+You can find all supported connectors and corresponding plugin_config configuration names under `${SEATUNNEL_HOME}/connectors/plugins-mapping.properties`.
+
+:::tip
```
Comment (TaoZex) [inline versioned_docs/version-2.3.0-beta/start-v2/local.mdx:62]: Is this a formatting problem？

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+./bin/start-seatunnel-flink-connector-v2.sh \
+--config ./config/seatunnel.streaming.conf.template
+```
+
+</TabItem>
+
+<TabItem value="SeaTunnel Engine">
+
+    ```shell
+    cd "apache-seatunnel-incubating-${version}"
```
Comment (TaoZex) [inline versioned_docs/version-2.3.0-beta/start-v2/local.mdx:163]: format error

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx (hunk 2 above)
Comment (EricJoy2048) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx:62]: > Is this a formatting problem？

Sorry, I can not open `.mdx` file, can you add a picture to let me know what's happened?

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx (hunk 3 above)
Comment (EricJoy2048) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx:163]: > format error

same as above.

Comment (TaoZex) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx:163]: shell

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx (hunk 2 above)
Comment (TaoZex) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx:62]: tip
We can use this to open the `.mdx` file:
mdx

Comment (EricJoy2048) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx:62]: > Is this a formatting problem？

I check finished. It's ok.

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx (hunk 1 above)
Comment (EricJoy2048) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx]: done

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx (hunk 2 above)
Comment (EricJoy2048) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx:62]: done

File: versioned_docs/version-2.3.0-beta/start-v2/local.mdx (hunk 3 above)
Comment (EricJoy2048) [reply versioned_docs/version-2.3.0-beta/start-v2/local.mdx]: done

**Review by TaoZex — APPROVED** [review]
LGTM

**Review by ic4y — APPROVED** [review]
LGTM

**Pull request merged**